packages/engines/toil/README.md

```diff
@@ -0,0 +1,51 @@
+## Toil AWS Mirror
+
+A Toil mono-container WES server for use with Amazon Genomics CLI.
+
+### Building the Container Manually
+
+Go to this directory and run:
+
+```bash
+docker build . -f Dockerfile -t adamnovak/toil-agc
```

Comment:
is this correct container image uri?

Reply:
That's not a tag I own on Docker Hub, but you can tag as that. The build procedure just needs an intermediate tag to avoid you always needing to use the long ECR repo tag.

I have some improvements to the Docker build procedure in https://github.com/adamnovak/amazon-genomics-cli/commit/26dc5add75f8091fcfbf012bfcc2eafc060548e7 where I fix a mistake in the example commands for the push, and also stop using grep to find stuff for local testing. I rebased that over onto here in c017154ee13261c3f4b93954dcad6a9ff7a41b37.